app/controllers/users_controller.rb

```diff
@@ -8,13 +9,26 @@ def create
     if @user.save
       redirect_to new_user_path
     else
-      render :new
+      render 'new'
     end
   end
 
-  def user_params
-    params.permit(:username, :email, :password_digest)
+  def edit
+    @user = User.find(params[:id])
   end
-  # {"username"=>"sdfs", "email"=>"asd@easdas", "password"=>"[FILTERED]", "authenticity_token"=>"wSniYW/JnuQ770uRAoeDaQ4YmwJiFdZZJJ6TW0taBaAzYNXullZhA/e+bowoXqGDuG2ngyUeFL7IfhmkUAkJ+A=="}
 
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
```

Comment:
Please, redirect to `edit_uer_path` if successful

Reply:
Fixed. Thank you.